examples/sql_bank/main.go

```diff
@@ -142,25 +155,31 @@ func verifyBank(db *sql.DB) {
 }
 
 func main() {
-	db, err := sql.Open("postgres", "dbname=postgres sslmode=disable")
+	security.SetReadFileFn(securitytest.Asset)
+	serv := server.StartTestServer(nil)
+	defer serv.Stop()
+
+	url := "https://root@" + serv.ServingAddr() + "?certs=test_certs"
```

Comment:
We should also be able to run it against arbitrary cockroach servers (e.g. a multi-node cluster), not just the in-process TestServer.